op-node/p2p/band_scorer_test.go

```diff
@@ -0,0 +1,57 @@
+package p2p
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestBandScorer_ParseDefault tests the [BandScorer.Parse] function
+// on the default band scores cli flag value.
+func TestBandScorer_ParseDefault(t *testing.T) {
+	defaultScoringBands := "-40:graylist;-20:restricted;0:nopx;20:friend;"
+
+	// Create a new band scorer.
+	bandScorer := NewBandScorer()
```

Comment:
See other comment, please also test `Bucket` and add an additional test of `Parse` that parsing fails if thresholds are not ordered from low to high.